hackgvl/hackgreenville-com
# Set TZ in transformer to America/New_York from EDT

# Summary
Switches the timezone used for casting event times in `EventDataTransformer` from using a type 2 "EDT" timezone to a type 3 "America/New_York" timezone that will automatically account for daylight savings time.

# Testing Plan
1. Before checking out this branch run `php artisan pull:events`
2. Launch the application and navigate to `/events`
3. The times, since we've fallen back from EDT to EST, will appear an hour later than they're actually scheduled for. Compare with the [OpenWorks events dashboard](https://joinopenworks.com/dashboard/meetups.php).
4. Also be sure to stop by the `/calendar` page to see that the same issue exists there.
5. Checkout the branch for this PR and re-run `php artisan pull:events`
6. Refresh the `/events` and `/calendar` pages. The times should now appear correctly.
